package.json

```diff
@@ -66,21 +82,36 @@
     "@folio/stripes": "^9.0.0",
     "@folio/stripes-cli": "^3.0.0",
     "@formatjs/cli": "^6.1.3",
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^14.0.0",
+    "@testing-library/user-event": "^14.4.3",
+    "@types/jest": "^29.5.1",
+    "@types/react": "^18.2.0",
+    "babel-jest": "^29.6.1",
     "core-js": "^3.6.1",
     "eslint": "^6.2.1",
     "eslint-plugin-filenames": "^1.3.2",
     "eslint-plugin-jest": "^23.0.4",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^29.6.1",
+    "jest-css-modules": "^2.1.0",
+    "jest-environment-jsdom": "^29.6.1",
+    "jest-junit": "^16.0.0",
     "faker": "^4.1.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-intl": "^6.4.4",
     "react-query": "^3.6.0",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
-    "regenerator-runtime": "^0.13.3"
+    "regenerator-runtime": "^0.13.3",
+    "ts-jest": "^28.0.7",
+    "typescript": "^4.7.4"
   },
   "dependencies": {
     "@folio/stripes-acq-components": "~5.0.0",
+    "@ngneat/falso": "^6.4.0",
```

Comment:
I didn't have a chance to see how it's used, but it's unpacked size is 935 kB, have you tested it with tree shaking (I even don't remember it's enabled in folio)

@zburke do we have any other alternatives?

Reply:
Previews are nice, but adding 1MB of data to support preview feels like too much. Is there another way to handle this?

Reply:
`import * as faker from '@ngneat/falso';` says a lot about usage, definitely should be optimized

Reply:
Hi @NikitaSedyx @zburke, if I were to specifiy specific functions in the import, does the optimize it enough? Shown in the first image. If not, I can hard code the function we use in a `fakerUtil.ts` file as shown in the second image, which is a basic structure and I need to still write the functions.

<img width="1008" alt="Screenshot 2024-02-04 at 8 50 37 PM" src="https://github.com/folio-org/ui-plugin-bursar-export/assets/97154067/f9146c17-7833-48fa-b8ac-25757e5de16f">
<img width="1096" alt="Screenshot 2024-02-04 at 8 53 14 PM" src="https://github.com/folio-org/ui-plugin-bursar-export/assets/97154067/ca957fde-1afe-41b7-8c48-df1150045735">


It is only used in the file `src/components/ExportPreview/createPreviewData.ts`.

Reply:
Agreed, @NikitaSedyx, that as we move toward lazy loading then this is less objectionable. Still it is a good reminder to be thoughtful about this kind of thing. Not every end user sits at the end of a fat connection adjacent to the hosting provider.

Reply:
@NikitaSedyx I understand. Do you have any examples of how to correctly lazy load a module like this? Is this along the right track?
<img width="1004" alt="Screenshot 2024-02-07 at 6 36 45 PM" src="https://github.com/folio-org/ui-plugin-bursar-export/assets/97154067/0360d61d-4b3f-443a-a9bb-02a71939407a">

Reply:
https://react.dev/reference/react/lazy

Reply:
@NikitaSedyx I am having trouble figuring out how to use lazy for this usage. It seems that lazy is used for React components, rather than a library. Does something like this dynamic import work? Or can you explain further how to use lazy here?
<img width="993" alt="Screenshot 2024-02-11 at 8 27 47 PM" src="https://github.com/folio-org/ui-plugin-bursar-export/assets/97154067/3126e252-0706-463f-bc68-add28990e0ef">

Reply:
you can have lazy component, or you can try dynamic import - not fully sure it works with our webpack, but you can try

Reply:
Ok, [here](https://github.com/folio-org/ui-plugin-bursar-export/pull/108#:~:text=import%20%27%40ngneat/falso%27-,f07ac99,-sonarcloud) is the implementation.